src/main/resources/com/google/api/codegen/java/common.snip

```diff
@@ -35,6 +35,27 @@
   @end
 @end
 
+@snippet commentedImportList(fileImports)
+  /*
+   * Please include the following imports to run this sample.
```

Comment:
Nit: Consider tweaking this to: "To run this sample, please include the following imports:" But I defer to @beccasaurus . It's fine as is, too.